src/blocks/numpy.ts

```diff
@@ -803,5 +803,252 @@ export const library: LibraryBlock[] = [
       },
     },
   },
+  {
+    name: "full",
+    uri: "numpy.full",
+    code: "from numpy import full",
+    returnType: "ndarray",
+    style: {
+      bg: { color: "yellow", opacity: 500 },
+      icon: "array-numeric",
+    },
+    arguments: [
+      {
+        name: "shape",
+        type: ["python.int", "python.int_array"],
+        description: "Shape of the new array, e.g., (2, 3) or 2",
+      },
+      {
+        name: "fill_value",
+        type: ["python.int", "python.int_array"],
+      },
+      {
+        name: "dtype",
+        type: ["numpy.type", "python.type"],
+        optional: true,
+        defaultValue: null,
+      },
+      {
+        name: "order",
+        type: "char",
+        optional: true,
+        defaultValue: "C",
+        options: [
+          {
+            label:
+              "Whether to store multidimensional data in C order in memory",
+            value: "C",
+          },
+          {
+            label:
+              "Whether to store multidimensional data in Fortran-contiguous order in memory",
+            value: "F",
+          },
+        ],
+      },
+      {
+        name: "array_like",
+        type: ["python.array", "numpy.array"],
+        optional: true,
+        defaultValue: "none",
```

Comment:
Sbagliato, nel caso in cui non ci sono valori di default, non mettere valori di default?